cmd/resource-topology-exporter/main.go

```diff
@@ -25,13 +23,25 @@ const (
 	ProgramName = "resource-topology-exporter"
 )
 
+type progArgs struct {
+	nrtupdater      *nrtupdater.Args
+	resourcemonitor *resourcemonitor.Args
+	rte             *resourcetopologyexporter.Args
+	version         *bool
```

Comment:
why do we need pointers here?

Reply:
Because ```flags.BoolVar()``` get a pointer and the sole purpose of the struct (which hold ```version```) is to use those ```flags.TypeVar()``` functions so it's more natural to pass ```version``` and not ```&version```

Reply:
I don't fully agree about version, but what about the other `Args`? Why do we need pointers to them?

Reply:
Now that I read this: https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/23#discussion_r691819551
I'll replace that.